package.json

```
@@ -66,7 +66,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/jackocnr/intl-tel-input.git"
+    "url": "https://github.com/Sonova-Marketing/intl-tel-input/"
```

are we using https here? wth ssh, it would be something like:

```suggestion
    "url": "git@github.com:Sonova-Marketing/intl-tel-input.git"
```

---

We don't ssh RN, we just login to the scope and publish. Yes, for ssh i think we will have to update.

@977mawais WDYT?

---

yes, you can update the url.

---

Resolving the conversation, as we agreed to use the https because there are no repeated deployments for this on.